pandora-common/src/assets/validation.ts

```diff
@@ -0,0 +1,67 @@
+import type { ItemId } from './appearanceTypes';
```

Comment:
This file seems to be missing in the `/src/assets/index.ts` file. Is that intentional?